browser/ui/views/frame/brave_browser_view.h

```diff
@@ -184,6 +184,10 @@ class BraveBrowserView : public BrowserView,
   bool IsActiveWebContentsTiled(const SplitViewBrowserData::Tile& tile) const;
   void UpdateSecondaryContentsWebViewVisibility();
 
+  // Returns the content::WebContents* for devtools.
```

Comment:
Maybe need to update this comment?

Reply:
Oops, right. We don't need the comment anymore.